resources/css/aphrodite-shortcomings.css

```diff
@@ -6,12 +6,79 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
+
 *, :before, :after {box-sizing: border-box;}
 strong {font-weight: bold;}
 
 /* See https://github.com/Khan/aphrodite/issues/30 */
+
 .episode-page {
   &__episode-transcript p {
     margin-bottom: 20px
   }
 }
+
+.container {
```

Comment:
Mostly, I'd just love to get rid of references to `container` in the components. I'd prefer to not have to have any `className`s in the components that are not aphrodite class names :-)